doc/radosgw/config-ref.rst

```diff
@@ -612,9 +612,9 @@ Swift Settings
 
 	      The ``X-History-Location`` attribute, also understood by
 	      OpenStack Swift for handling ``DELETE`` operations
-	      `slightly differently
-	      <https://docs.openstack.org/swift/latest/overview_object_versioning.html>`_
-	      from ``X-Versions-Location``, is currently not
+	      `(slightly different
```

Comment:
probably, instead, we can link "OpenStack Swift" to https://docs.openstack.org/swift/latest/api/object_versioning.html. like
```rst
A slightly different attribute, ``X-History-Location``, which is also understood by 
`OpenStack Swift <https://docs.openstack.org/swift/latest/api/object_versioning.html>`
for handling ``DELETE`` operations, is currently not supported.
```

Reply:
@tchaikov yeah, that sounds much better :)

Reply:
@tchaikov I made the change.